### ISSUE TITLE
Code cleanup: addcto removeCommand (and maybe others)

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3624]
I added `refreshOnDescendantChange` logic to AddCommand and ModifyCommand because it was needed for dojo mobile ListItem widgets. For completeness, it should be added to RemoveCommand (even though not needed for this particular scenario). Also, need to look at other commands to see if they need this logic too.

Also, right now the code is duplicated in AddCommand and ModifyCommand. Should factor out into a base class or mixin.

**Comment by JonFerraiolo** [comment]
Started working on this, but was blocked by #3742
